### PR TITLE
Began reporting client sdk information

### PR DESCRIPTION
**Pull request by maneesht**

### CODE REVIEW

**Review by hsubox76 — COMMENTED** [review]
I think you might want to merge master into the main dataconnect branch? It looks like you merged master into this branch but not the main dataconnect branch so there's a really big diff that includes a lot of unrelated changes?

**Comment by maneesht** [comment]
> I think you might want to merge master into the main dataconnect branch? It looks like you merged master into this branch but not the main dataconnect branch so there's a really big diff that includes a lot of unrelated changes?

Ah, good point. Done, thanks

File: docs-devsite/index.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ https://github.com/firebase/firebase-js-sdk
 |  [@firebase/app](./app.md#app_package) | Firebase App |
 |  [@firebase/app-check](./app-check.md#app-check_package) | The Firebase App Check Web SDK. |
 |  [@firebase/auth](./auth.md#auth_package) | Firebase Authentication |
+|  [@firebase/data-connect](./data-connect.md#data-connect_package) | Firebase Data Connect |
```
Comment (markarndt) [inline docs-devsite/index.md:22]: Is docs-devsite/data-connect.md in another repo?

Comment (maneesht) [reply docs-devsite/index.md:22]: It's in this branch I believe

**Pull request merged**